cmd/helm/install.go

```diff
@@ -151,6 +152,7 @@ func addInstallFlags(f *pflag.FlagSet, client *action.Install, valueOpts *values
 	f.BoolVar(&client.Atomic, "atomic", false, "if set, installation process purges chart on fail. The --wait flag will be set automatically if --atomic is used")
 	f.BoolVar(&client.SkipCRDs, "skip-crds", false, "if set, no CRDs will be installed. By default, CRDs are installed if not already present")
 	f.BoolVar(&client.SubNotes, "render-subchart-notes", false, "if set, render subchart notes along with the parent")
+	f.BoolVar(&client.ReadOnlyFS, "readonly-fs", false, "if set, do not download charts to the HELM_REPOSITORY_CACHE")
```

Comment:
This flag name doesn't seem to match what it does. I mean, I could run this on a read-write filesystem just as easily, right?

Reply:
I'm glad you brought this up. I'm having a lot of trouble coming up with a future-proof name, for the flag but also for the corresponding API in #7601.

Reply:
@technosophos Yes you should be able to run this on a read write file system just easily.